Foods.yaml

```
@@ -6,7 +6,4 @@
 
 FavouriteFoods:
   # Add the food to this list on this line, if possible.
-
-
-  - ["pizza", "kale_smoothies", "bacon", "cheese", "pie", "carbonara", "garlic_bread", "banana", "burrito", "coffee", "chocolate", "chicken_schnitzel", "mango", "macadamias", "Tacos"]
-
+  - ["pizza", "kale_smoothies", "bacon", "cheese", "pie", "carbonara", "garlic_bread", "banana", "burrito", "coffee", "chocolate", "chicken_schnitzel", "mango", "macadamias", "tacos (mince & cheese"]
```

```suggestion
  - ["pizza", "kale_smoothies", "bacon", "cheese", "pie", "carbonara", "garlic_bread", "banana", "burrito", "coffee", "chocolate", "chicken_schnitzel", "mango", "macadamias", "tacos (mince & cheese)"]
```
You forgot to close the parentheses :stuck_out_tongue: